app/controllers/concerns/pageable_concern.rb

```diff
@@ -0,0 +1,8 @@
+module PageableConcern
+  extend ActiveSupport::Concern
+
+  def items_per_page
+    # In case we want to make this customizable from the page later.
+    params[:items_per_page] || Kaminari.config.default_per_page
+  end
+end
```

Comment:
Concern that allows us, on pages that have paging of some sort, to implement a (for example) select control to alter the items per page returned - just thinking about enhancements.